_includes/head.html

```diff
@@ -30,4 +30,7 @@
 
     <!-- longlist -->
     <script src={{ "/js/longlist.min.js" | prepend: site.baseurl }}></script>
+
+    <!-- ical -->
+    <script src={{ "/js/ical.min.js" | prepend: site.baseurl }}></script>
```

Comment:
kann doch eigtl vor den script block in {{_includes/treffen.md}}? Wird ja sonst auf jeder Seite geladen